### ISSUE TITLE
REBUILD_ALL needs to be present

### ISSUE THREAD

**Issue by jordpo** [tonycoco/heroku-buildpack-ember-cli#118]
We're running into an issue where our Heroku deploy fails every time if the REBUILD_ALL is not set to true. Here is the failed deploy logs

```
-----> Fetching set buildpack https://github.com/tonycoco/heroku-buildpack-ember-cli.git... done
-----> Ember CLI app detected
-----> Exporting config vars to environment
-----> Building environment: production
       PRO TIP: Avoid using semver ranges starting with '>' in engines.node
       See https://devcenter.heroku.com/articles/nodejs-support
-----> Requested node range:  >= 0.10.0
-----> Resolved node version: 5.1.1
-----> Downloading and installing node
-----> Using default npm version: 3.3.12
-----> Downloading and installing nginx
-----> Adding boot script
-----> Copying configs
-----> Restoring node_modules directory from cache
-----> Restoring bower_components directory from cache
-----> Bower already exists
-----> Pruning cached dependencies not specified in package.json
       unbuild ansi-escapes@1.1.0
       unbuild archy@1.0.0
       unbuild array-filter@0.0.1
       unbuild array-map@0.0.0
       unbuild array-reduce@0.0.0
       unbuild binary@0.3.0
       unbuild boom@0.4.2
       unbuild bower-logger@0.2.2
       unbuild buffers@0.1.1
       unbuild caseless@0.8.0
       unbuild chainsaw@0.1.0
       unbuild chmodr@0.1.0
       unbuild cli-color@0.3.3
       unbuild cli-cursor@1.0.2
       unbuild cli-width@1.1.0
       unbuild code-point-at@1.0.0
       unbuild cryptiles@0.2.2
       unbuild d@0.1.1
       unbuild deep-extend@0.2.11
       unbuild end-of-stream@1.1.0
       unbuild es5-ext@0.10.10
       unbuild es6-iterator@2.0.0
       unbuild es6-symbol@3.0.2
       unbuild event-emitter@0.3.4
       unbuild exit-hook@1.1.1
       unbuild figures@1.4.0
       unbuild fstream-ignore@1.0.3
       unbuild hawk@1.1.1
       unbuild hoek@0.9.1
       unbuild imurmurhash@0.1.4
       unbuild infinity-agent@2.0.3
       unbuild inherit@2.2.2
       unbuild intersect@0.0.3
       unbuild is-fullwidth-code-point@1.0.0
       unbuild is-npm@1.0.0
       unbuild is-redirect@1.0.0
       unbuild is-root@1.0.0
       unbuild is-stream@1.0.1
       unbuild junk@1.0.2
       unbuild lockfile@1.0.1
       unbuild lodash.debounce@3.1.1
       unbuild lowercase-keys@1.0.0
       unbuild lru-queue@0.1.0
       unbuild memoizee@0.3.9
       unbuild mime-types@1.0.2
       unbuild minimist@0.0.10
       unbuild mkpath@0.1.0
       unbuild mout@0.11.1
       unbuild mute-stream@0.0.4
       unbuild nested-error-stacks@1.0.2
       unbuild next-tick@0.2.2
       unbuild oauth-sign@0.5.0
       unbuild onetime@1.0.0
       unbuild optimist@0.6.1
       unbuild package-json@1.2.0
       unbuild pinkie@1.0.0
       unbuild pinkie-promise@1.0.0
       unbuild prepend-http@1.0.3
       unbuild promptly@0.2.0
       unbuild pump@1.0.1
       unbuild read@1.0.7
       unbuild redeyed@0.4.4
       unbuild registry-url@3.0.3
       unbuild request-progress@0.3.1
       unbuild request-replay@0.2.0
       unbuild restore-cursor@1.0.1
       unbuild retry@0.6.1
       unbuild run-async@0.1.0
       unbuild rx@2.5.3
       unbuild rx-lite@3.1.2
       unbuild shell-quote@1.4.3
       unbuild slide@1.1.6
       unbuild sntp@0.2.4
       unbuild string-length@1.0.1
       unbuild stringify-object@1.0.1
       unbuild tar-fs@1.8.1
       unbuild throttleit@0.0.2
       unbuild timed-out@2.0.0
       unbuild timers-ext@0.1.0
       unbuild tmp@0.0.24
       unbuild traverse@0.3.9
       unbuild update-notifier@0.3.2
       unbuild uuid@2.0.1
       unbuild wordwrap@0.0.3
       unbuild xdg-basedir@1.0.1
       unbuild bower-config@0.6.1
       unbuild bower-json@0.4.0
       unbuild bower-registry-client@0.3.0
       unbuild configstore@0.3.2
       unbuild duplexify@3.4.2
       unbuild es6-weak-map@0.1.4
       unbuild got@3.3.1
       unbuild insight@0.7.0
       unbuild p-throttler@0.1.1
       unbuild read-all-stream@3.0.1
       unbuild readline2@0.1.1
       unbuild semver-diff@2.1.0
       unbuild tar-stream@1.3.1
       unbuild touch@0.0.3
       unbuild win-release@1.1.1
       unbuild write-file-atomic@1.1.4
       unbuild inquirer@0.8.0
       unbuild bower@1.5.3
       unbuild cardinal@0.4.4
       unbuild esprima@1.0.4
       unbuild latest-version@1.0.1
       unbuild opn@1.0.2
       unbuild os-name@1.0.3
       unbuild decompress-zip@0.1.0
       unbuild handlebars@2.0.0
       unbuild js-yaml@3.4.6
       unbuild osx-release@1.1.0
       unbuild rc@1.1.5
-----> Pruning cached bower dependencies not specified in bower.json
       /tmp/buildpack20151215-159-x8a6lu/bin/compile: line 186: bower: command not found
 !     Push rejected, failed to compile Ember CLI app
```

**Comment by tonycoco** [comment]
Maybe fixed with #123... please verify.

**Comment by jordpo** [comment]
@tonycoco yeah this is fixed. we created a new project and didn't run into this issue.
